deploy/olm-catalog/ibm-block-csi-operator-community/1.8.0/manifests/ibm-block-csi-operator.v1.8.0.clusterserviceversion.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     description: "Run IBM block storage CSI driver."
     repository: https://github.com/IBM/ibm-block-csi-operator
     support: IBM
-    alm-examples: >-
+    alm-examples: |-
```

Comment:
was this file changed manually or automatically?

Reply:
this change is manually